.gitattributes

```diff
@@ -1,5 +1,6 @@
+/.github export-ignore
+/docs export-ignore
```

Comment:
These should still be exported, IMO

Reply:
sure fine for me. it wasn't exported once before due to /html-docs export-ignore. i guess the /examples should be removed from the gitattributes as well?

Reply:
Yes please! HTML docs are long gone. Thanks for looking into this!

Reply:
done